docs/SQL/gems/transformations/deduplicate.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Deduplicate
+id: deduplicate
+description: Remove rows with duplicate values of specified columns
+sidebar_position: 6
+tags:
+  - gems
+  - dedupe
+  - distinct
+  - unique
+---
+
+Removes rows with duplicate values of specified columns.
+
+## Parameters
+
+| Parameter              | Description                                                                                                                                                                                                                                                                                                                | Required |
+| :--------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------- |
+| Source                 | Input source                                                                                                                                                                                                                                                                                                               | True     |
+| Row to keep            | - `Distinct Rows`: Keeps all distinct rows. This is equivalent to performing a `df.distinct()` operation <br/>- `Unique Only`: Keeps rows that don't have duplicates <br/>- `First`: Keeps first occurrence of the duplicate row <br/>- `Last`: Keeps last occurrence of the duplicate row <br/>Default is `Distinct Rows` | True     |
```

Comment:
`df.distinct()` is for Spark. We can say equivalent to performing `select distinct ...`

Reply:
Done.